### ISSUE TITLE
[beta.4] Shouldn't events have unique ids?

### ISSUE THREAD

**Issue by superandrew213** [fullcalendar/fullcalendar#4499]
It says in the docs that event.id `Will uniquely identify your event. Leveraged by getEventById.`

Is this still WIP?

See codepen below. There are 2 events with same id. Calling `getEventById` returns first event only.

https://codepen.io/anon/pen/zePWwM?editors=0010

**Comment by acerix** [comment]
It seems to be working as documented, are you suggesting there should be an error message in this case?

**Comment by superandrew213** [comment]
This is my actual issue: https://github.com/fullcalendar/fullcalendar/issues/4502

If we say `Will uniquely identify your event`, then uniqueness should be enforced. An error should be thrown if there are duplicate events.

Since `id` is not a required field, this can be restricted to events that have an id only.

**Comment by arshaw** [comment]
I agree, a console error should probably be displayed, and the event should not be accepted in this case

**Comment by arshaw** [comment]
I'm gonna punt on this for now because there are too many touchpoints in the code for when events are added to the internal event stores.
